tests/flytekit/unit/core/test_data_persistence.py

```diff
@@ -14,3 +14,8 @@ def test_is_remote():
     assert fp.is_remote("/tmp/foo/bar") is False
     assert fp.is_remote("file://foo/bar") is False
     assert fp.is_remote("s3://my-bucket/foo/bar") is True
+
+
+def test_lister():
+    x = DataPersistencePlugins.supported_protocols()
+    assert [y.replace("://", "") for y in x] == ["file", "/", "gs", "http", "https", "s3"]
```

Comment:
@pingsutw can you help me take a look at this test?  It's failing when I have the fsspec plugin installed because the fsspec plugin registers itself with all these protocols:
```
['file',
 '/',
 'gs',
 'http',
 'https',
 's3',
 'memory',
 'dropbox',
 'zip',
 'tar',
 'gcs',
 'gdrive',
 'sftp',
 'ssh',
 'ftp',
 'hdfs',
 'arrow_hdfs',
 'webhdfs',
 's3a',
 'wandb',
 'oci',
 'adl',
 'abfs',
 'az',
 'cached',
 'blockcache',
 'filecache',
 'simplecache',
 'dask',
 'dbfs',
 'github',
 'git',
 'smb',
 'jupyter',
 'jlab',
 'libarchive',
 'reference',
 'generic',
 'oss',
 'webdav']
```
Does it really handle all that?!  There's no way right?

Reply:
I think Fsspec can handle all of [them](https://filesystem-spec.readthedocs.io/en/latest/api.html#built-in-implementations). 
Could we change it to
`assert {"file", "/", "gs", "http", "https", "s3"}.issubset(set([y.replace("://", "") for y in x]))`